aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,41 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## January 2023
+
+- **Page Templates**
+  - Began identifying and exploring a variety of different types of common PageLayouts to create Page Templates. Coming soon!
+- **HPE Global Header**
+  - [Global Header](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?t=rcDz07wh34Xxtsft-0) available in Figma now.
+- **Feedback**
+  - [Guidance](https://design-system.hpe.design/templates/user-feedback-collection?q=feed) was added to the User Feedback Collection page.
+ - **Skeleton**
+[Skeleton example and guidance](https://design-system.hpe.design/components/skeleton?q=skele) is now available.
+- **Data**
+  - Added a how to add [search and filter to DataTable with Data guide](https://design-system.hpe.design/learn/how-to-add-search-and-filter-to-datatable-with-data?q=data).
+- **Figma Refactored Components:**
+  - [CheckBox](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2940%3A62424&t=da2J4GDdKdAi7PIN-1)
+  - [CheckBoxGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5732%3A78041&t=da2J4GDdKdAi7PIN-1)
+  - [RadioButton](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=3520%3A83485&t=da2J4GDdKdAi7PIN-1)
+  - [RadioButtonGroup](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5733%3A92878&t=da2J4GDdKdAi7PIN-1)
+  - [Tip](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5248%3A73430&t=da2J4GDdKdAi7PIN-1)
+  - [Feedback](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5554%3A77765&t=da2J4GDdKdAi7PIN-1)
+
+## December 2022
+
+- **Layer**
+  - Layer- New [refactored Layer components are available in Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5735%3A85900&t=da2J4GDdKdAi7PIN-1
```

Comment:
```suggestion
  - New [refactored Layer components are available in Figma](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=5735%3A85900&t=da2J4GDdKdAi7PIN-1
```
I don't think we need to have layer again since we have it bolded directly above